builders/get-started/networks/layer2/bobabase.md

```diff
@@ -0,0 +1,76 @@
+
+<div class="button-wrapper">
+    <a href="#" class="md-button connectMetaMask" value="bobabase">Connect MetaMask</a>
+</div>
+
+!!! note
+    MetaMask will popup asking for permission to add Bobabase as a custom network. Once you approve permissions, MetaMask will switch your current network to Bobabase.
+
+If you do not have MetaMask installed, or would like to follow a tutorial to get started, please check out the [Interacting with Moonbeam using MetaMask](/tokens/connect/metamask/) guide.
+
+## Get Tokens with the Faucet {: #get-tokens-with-the-faucet } 
+
+Testnet BOBA is distributed via a Tweet-authenticated faucet on [Bobabase Gateway](https://gateway.bobabase.boba.network/){target=_blank}, a home for your Boba Layer 2 activities akin to [Apps.Moonbeam.Network](https://apps.moonbeam.network/){target=_blank}. Bobabase allows fee payment in either DEV or testnet BOBA, and you can toggle this setting via a dropdown in the upper right corner.
+
+To get some testnet BOBA, take the following steps:
+
+1. Head to [Bobabase Gateway](https://gateway.bobabase.boba.network/){target=_blank}
+2. Ensure you're on the [BobaBase Network](#connect-metamask) and Click **Connect** to Connect your MetaMask Wallet
+3. Press **Tweet Now**, and Send the Public Tweet
+4. Copy the link to the tweet and paste it in the box 
+5. Press **Authenticated Faucet** and sign the following MetaMask popup
+
+![Bobabase Faucet](/images/builders/get-started/networks/bobabase/bobabase-1.png)
+
+If you don't have a Twitter account, you can contact us on [Discord](https://discord.gg/PfpUATX){target=_blank} to receive testnet BOBA.
+
+## Bridge from Moonbase Alpha to Bobabase {: #bridge-from-moonbase-alpha-to-bobabase }
+
+[Bobabase Gateway](https://gateway.bobabase.boba.network/){target=_blank} enables you to bridge various assets to and from Bobabase. To bridge assets from Moonbase Alpha to Bobabase, take the following steps:
+
+1. Head to [Bobabase Gateway](https://gateway.bobabase.boba.network/){target=_blank} and Press **Connect**
+2. Click on **Moonbase Wallet** in the upper left corner 
+3. Next to the asset you'd like to bridge, press **Bridge to L2**
+4. Enter the amount you'd like to bridge and press **Bridge**
+5. Confirm the transaction in MetaMask
+
+![Bridge to Bobabase](/images/builders/get-started/networks/bobabase/bobabase-2.png)
+
+## Bridge from Bobabase to Moonbase Alpha {: #bridge-from-bobabase-to-moonbase-alpha }
+
+[Bobabase Gateway](https://gateway.bobabase.boba.network/){target=_blank} enables you to bridge various assets to and from Bobabase. Note, when bridging back from the Layer 2, there is a 7-day delay before your funds are available on Moonbase Alpha. This delay is an inherent safety feature of the optimistic rollup architecture and applies only when bridging from the Layer 2 back to the Layer 1. To bridge assets from Bobabase to Moonbase Alpha, take the following steps:
```

Comment:
I think the 7-day delay should be a variable, as it could potentially change in the future (right? haha) and if it did we would only have to update the variables.yml file instead of both english and chinese repo